VespaGAS/ViewModels/GasCalculatorViewModel.swift

```diff
@@ -1,5 +1,36 @@
 import Foundation
+import RxSwift
 
-public struct GasCalculatorViewModel {
+struct GasCalculatorViewModel {
+    private let disposeBag = DisposeBag()
+    private let oilValueSubject = BehaviorSubject(value: "0%")
```

Comment:
The reason why I'm using a BehaviorSubject is that it can be initialised with an initial value. 
Is it a good idea?